docs/index.rst

```diff
@@ -8,6 +8,9 @@ GeoCAT-tutorials
 Welcome to GeoCAT-tutorials! This version of this content was made for 
 presentation at the Fall 2022 ESDS event.
 
```

Comment:
Small suggestion: Could we make this more obvious on the webpage using a note admonition? It's not a necessity, but I just thought of it.

Reply:
@hCraker Oh gosh I missed your comment before I merged, sorry! Two things:
- this would have had to have been done before I tagged the release, but is definitely something we should consider doing in the future!
- the docs should display a version warning admonition, though it won't be as clean as the text you suggested above

Reply:
No worries! It's not a crucial addition. It's mostly something to keep in mind for future events